motr/m0crate/crate_index.c

```diff
@@ -225,6 +228,8 @@ struct cr_time_measure_ctx {
 	m0_time_t	ts;
 	/** Last timestamp. */
 	m0_time_t	ts_next;
+	/** Total test timestamp. */
+	m0_time_t	test_time;
```

Comment:
Seem we don't need this since you can begin and end and get diff

Reply:
Fixed.